Simple calory app

kiindmanjio opened the pull request:
#WHAT 
developed a simple calory calculation app.

#WHY 
In order to make it simple for those who want to calculate the caloric input.

Commit:
modified gets method

Commit:
developed calory app

offich commented on ruby.rb:
```diff
@@ -1,2 +1,54 @@
+  end
+
+  def name
+    return @name
```
rubyでは、returnは書かないようにしよう。

kiindmanjio replied:
そうなの?!笑
rubyのルール??

offich replied:
そうだね。受講生には、わかりやすく、returnを使うことを教えているけど、rubyはreturn いらない言語仕様だから、覚えておいて笑

offich commented on ruby.rb:
```diff
@@ -1,2 +1,54 @@
+  end
+
+  def calory
+    return @calory
```
ここでも

offich commented on ruby.rb:
```diff
@@ -1,2 +1,54 @@
+  end
+
+
+  def initialize(name,calory)
```
initialize メソッドは、一番上にかこう

kiindmanjio replied:
これもルール???

offich replied:
単純に、コンストラクタは、一番上に書かれているから笑
英語でいうと、He am って言わないでしょ？笑

offich reviewed (changes requested):
レビュしました〜

offich commented:
あと、PRの概要で、# のマークは、スペースを空けるようにしよう。

Commit:
modified simple_calory_app. deleted return and changed the initialize method to the upmost.

kiindmanjio commented:
# WHAT
 -deleted return 
 -Changed the position of initialize method to the uppermost of Food class.
# WHY 
Is it because it is the rule of ruby??
if would be great if you can answer my question.

offich commented:
↑　上のwhatとかwhyは、一番上を更新すれば、大丈夫。
質問はこんな感じで、コメントしてくれれば、いいから。

kiindmanjio commented:
なるほど了解です！

r-ume commented on ruby.rb:
```diff
@@ -1,2 +1,55 @@
+
+    @@foods.each do |food|
+      puts "#{food.name} :#{food.calory}kcal"
+      all_calory = all_calory + food.calory
```
これは、 `all_calory += food.calory`って書こうか。

offich replied:
これは、`all_calory += food.calory` って書こうか

offich commented on ruby.rb:
```diff
@@ -1,2 +1,55 @@
+    puts "#{all_calory}"
+  end
+
+
```
余行は削除しましょ

offich reviewed (changes requested):
あとちょっと〜

Commit:
deleted extra lines

Commit:
abbreviate line 26

kiindmanjio commented:
<質問>
・return を使わないのはrubyのルール??
・initializeメソッドを初めに書くのはrubyのルール??

修正done
お願いします！！

offich commented:
あ、上の質問の答えは、じおが質問したところに書いてる

kiindmanjio commented:
おー！確認できた！

offich commented:
LGTM
